### ISSUE TITLE
Hyperparameter for Triplet-Loss

### ISSUE THREAD

**Issue by melgor** [KevinMusgrave/powerful-benchmarker#2]
Hi,
I'm investigating further your code/methodology of testing. Then I have some question.
You decided to choose margin for triplet just `0.01` when the default value is `0.2`. 
Any reason/paper for that?  Did you get this value after some experiments or so?

**Comment by KevinMusgrave** [comment]
Yes I tried out a few different values for the margin. It seems like small margins work best (assuming the embeddings are normalized). My guess is that large margins cause overfitting.

**Comment by KevinMusgrave** [comment]
In my latest experiments, I've been using bayesian optimization to find hyperparameters that perform well during cross-validation. It seems like a margin closer to 0.1 (not 0.01) works better.

**Comment by ssbilakeri** [comment]
Hi 
can you please suggest me one paper or give your test values  where different margin values are tested ? 
could you please explain how small margin works better

**Comment by KevinMusgrave** [comment]
You can see the hyperparameters here: https://kevinmusgrave.github.io/powerful-benchmarker/papers/mlrc/#optimization-plots

My understanding is that a large triplet margin causes overfitting.

**Comment by ssbilakeri** [comment]
Thank you for your response
